keras_cv/backend/__init__.py

```diff
@@ -42,6 +42,8 @@
 
 if multi_backend():
     import keras_core as keras
+
+    keras.backend.name_scope = keras.src.backend.name_scope
```

Comment:
this looks like it is exported as `keras.name_scope`, can we just use it like that?

Reply:
https://github.com/keras-team/keras-core/blob/b35cdfb18d7b49ff40f294943be4fefb10f0c85e/keras_core/backend/exports.py#L27

Reply:
Oh awesome! Yes we can -- this wasn't available when I wrote this PR.